neo4j-ogm-docs/src/main/asciidoc/reference/getting-started.adoc

```diff
@@ -11,7 +11,7 @@ Consult the version table to determine which version of Neo4j-OGM to use with a
 [frame="topbot",options="header"]
 |======================
 |Neo4j-OGM Version   |Neo4j Version^1^
-|3.2.x               |3.2.x, 3.3.x, 3.4.x, 3.5.x
+|3.2.x               |3.2.x, 3.3.x, 3.4.x, 3.5.x, 4.0.x, 4.1.x
```

Comment:
```suggestion
|3.2.x               |3.2.x, 3.3.x, 3.4.x, 3.5.x
```

This change suggests that OGM 3.2 can bring up Neo4j embedded in that versions. This is not possible.
I'm happy to add those versions but as a separate paragraph pointing out that this is only supported over bolt, either a real (remote) instance or an embedded instance via local socket connection.

Reply:
Let me know if 54a5d564a8729bd71075df64d2816169c41e6ba7 is not enough